msgpack.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: msgpack
+  version: 3.3.0
+  epoch: 0
+  description: "msgpack for C/C++."
+  copyright:
+    - license: Boost Software License - Version 1.0
```

Comment:
@kaniini are we concerned about this license?

Reply:
No, its a free license, its fine.  Use SPDX tag `BSL-1.0`.